src/main/java/nl/knaw/dans/pf/language/ddm/relationhandlers/DdmIsVersionOfHandler.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2014 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.pf.language.ddm.relationhandlers;
+
+import nl.knaw.dans.pf.language.ddm.handlertypes.BasicIdentifierHandler;
+import nl.knaw.dans.pf.language.emd.types.BasicIdentifier;
+import nl.knaw.dans.pf.language.emd.types.Relation;
+import org.xml.sax.SAXException;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class DdmIsVersionOfHandler extends BasicIdentifierHandler {
+    @Override
+    public void finishElement(final String uri, final String localName) throws SAXException {
+        final BasicIdentifier relation = createIdentifier(uri, localName);
+        final String href = getAttribute("", "href");
+
+        if (href == null)
+            throw new SAXException("href attribute is mandatory in ddm:isVersionOf");
```

Comment:
well, according to the ddm.xsd it is not mandatory. do you really want to enforce the distinction between dc:isVersionOf and ddm:isVersionOf wrt this attribute?